airy.go

```diff
@@ -0,0 +1,29 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import "github.com/gonum/mathext/internal/amos"
+
+// Airy returns the Airy function at z. The Airy function here, Ai(y), is one
```

Comment:
I would rewrite this either as
```
// Airy returns the value of the Airy function at z. The Airy function is one
// of the two linearly independent solutions to
//  y'' - y*z = 0.
// See http://mathworld.wolfram.com/AiryFunctions.html for more detailed information.
```
or simply as
```
// Airy returns the value of the Airy function at z.
// See http://mathworld.wolfram.com/AiryFunctions.html for more detailed information.
```
In other words:
- It does not return the function, it returns its value.
- It's probably not necessary to denote the function as Ai. If you think it is, then the argument should be z, not y.
- With the url to more information, it's probably not even necessary to say how the Airy function is defined. Up to you.